### PR TITLE
Allow "." in queryParam and queryParamRaw func

### LINKED ISSUE #1115: "." in parameter name not working for queryParam and queryParamRaw
**Describe the bug**
[PR 768](https://github.com/mockoon/mockoon/pull/768) fixed similar issue in helper functions body and bodyRaw but not in queryParam and queryParamRaw

**To Reproduce**
Use queryParam param.name and it won't work.

I am proposing a fix in request-helpers.ts to add code similar to body and bodyRaw functions to fix queryParam and queryParamRaw functions.

Let me know if I can raise a PR to fix this or has this been left out on purpose in PR 768 for some known issue?

### PR DESCRIPTION
**Pull request by ajatkj**
**Technical implementation details**

Added call to function convertPathToArray for queryParam and queryParamRaw function.

Closes #1115

### CODE REVIEW

**Comment by 255kb** [comment]
Thank you for opening the PR. 
Could you please add a test case for [each helper](https://github.com/mockoon/mockoon/blob/476eaea91ac4623ed4e8ca50b024b9fc25a9a00f/packages/commons-server/test/suites/templating-helpers/request-helpers.spec.ts#L294) like it was done for `body` and `bodyRaw`? 
Thanks

File: packages/commons-server/test/suites/templating-helpers/request-helpers.spec.ts [hunk 1]
```diff
@@ -507,6 +518,16 @@ describe('Request helpers', () => {
       );
       expect(parseResult).to.be.equal('dolphin');
     });
+    it('should return the value of parameter with dots in parameter name', () => {
+      const parseResult = TemplateParser(
+        false,
+        "{{queryParam 'param1\\.name' undefined true}}",
```
Comment (255kb) [inline packages/commons-server/test/suites/templating-helpers/request-helpers.spec.ts:524]: shouldn't it be a test with `queryParamRaw` instead?

**Review by 255kb — CHANGES_REQUESTED** [review]

File: packages/commons-server/test/suites/templating-helpers/request-helpers.spec.ts (hunk 1 above)
Comment (ajatkj) [reply packages/commons-server/test/suites/templating-helpers/request-helpers.spec.ts:524]: Damn! I missed that. Thanks for catching it. Sorry about oversight.

Comment (255kb) [reply packages/commons-server/test/suites/templating-helpers/request-helpers.spec.ts:524]: No problem!

Comment (ajatkj) [reply packages/commons-server/test/suites/templating-helpers/request-helpers.spec.ts:524]: There's same problem on test above it from where I copied. I will fix that as well.

Comment (255kb) [reply packages/commons-server/test/suites/templating-helpers/request-helpers.spec.ts:524]: Ahah, indeed. Thanks for fixing my bad code!

Comment (ajatkj) [reply packages/commons-server/test/suites/templating-helpers/request-helpers.spec.ts]: No problem!
Done. Hope everything is fine now :)

Comment (255kb) [reply packages/commons-server/test/suites/templating-helpers/request-helpers.spec.ts]: Thanks, just in time for the release :)

**Review by 255kb — APPROVED** [review]

**Pull request merged**